_episodes/02-filedir.md

```diff
@@ -469,7 +469,9 @@ which doesn't exist.
 > ```
 > {: .language-bash}
 > `ls` is the command, `-lh` are the flags (also called options),
-> and `Documents` is the argument.
+> and `Documents` is the argument.  Notice in this example, we combined two
+separate flags, `-l -h` into a single flag, `-lh`.  This grouping is common
```

Comment:
`-lh`  isn't a single flag, it's still two flags, just grouped together.

Reply:
I think the meaning is clear, but you're right, the wording is imprecise. If you have a suggestion for how to reword this, a PR would be welcome.